content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,243 @@
+kubectl apply -f alpine-gitsource.yaml -f alpine-kustomization.yaml
+```
+
+or you can use the [Flux CLI](https://fluxcd.io/docs/cmd/#installation) to do the same thing.
+
+To create `GitRepository`:
+
+```shell
+flux create source git alpine \
+    --url https://github.com/developer-guy/desired-state-repository \
+    --branch master \
+    --interval 30s
+```
+
+To create `Kustomization`:
+
+```shell
+flux create kustomization alpine \
+    --source alpine \
+    --path "./" \
+    --prune true \
+    --validation client \
+    --interval 5m
+```
+
+### Install falco-event-listener
+
+As the name suggests, this project will listen to an event in form of `CloudEvents` that forwarded from Falcosidekick which is a simple daemon for enhancing available outputs for Falco. Once, an event received from Falcosidekick, this project will update the YAML definition to scale its replicas to zero based on the informations in the event.
+
+At the end of the day, the architecture of the demo is going to be:
+
+Falco w/HTTP --> Falcosidekick w/CloudEvent --> falco-event-listener
+
+If you want to learn more about `CloudEvents`, you can follow this [link](https://cloudevents.io).
+
+Let's clone the project's repository first.
+
+```shell
+git clone https://github.com/developer-guy/falco-event-listener
+cd falco-event-listener
+```
+
+This project is basically a CLI application and uses the following [arguments](https://github.com/developer-guy/falco-event-listener/blob/master/falcoeventlistener.yaml#L12):
+
+```yaml
+args:
+  - "--owner"
+  - "developer-guy" # this is the owner of the desired state repository
+  - "--repository"
+  - "desired-state-repository" # this is the repository which we store desired state configurations
+  - "--file"
+  - "alpine.yaml" # this is the file that we are going to update
+  - "--github-token"
+  - "$(GITHUB_TOKEN)"
+```
+
+Before installing this project, we have to [create a GitHub PAT (Personal Access Token)](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to update the desired state configurations which is in GitHub repository after we detect a malicious behaviour related to our Pod.
+
+As you can see in the above arguments, we should pass `GITHUB_TOKEN` as an argument to our CLI application, in order to do that in Kubernetes environment, the best option is storing `GITHUB_TOKEN` in a Kubernetes Secret and [using Secret as environment variables](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables).
+
+```shell
+kubectl create namespace falco
+kubectl create secret generic github-secret \
+         --from-literal GITHUB_TOKEN=$GITHUB_TOKEN \
+         --namespace falco
+```
+
+Last but least, we're going to use `ko` tool which is created by Google to build, push our container image and deploy the project on Kubernetes cluster. `ko` is a simple, fast container image builder for Go applications.
```

Comment:
Finally,  we will use the `ko` tool to build and push our container image and deploy the application on the Kubernetes cluster. `ko`, created by Google, is a simple, fast container image builder for Go applications.

Reply:
thank you so much for fixing the issues, done.